docs/modules/ROOT/pages/how-to-guides/Import-code/proc_importing_code.adoc

```diff
@@ -226,8 +226,8 @@ Create the Red Hat Container Registry token by following these steps:
 . Create a registry service account by clicking *New Service Account*. 
 . Fill in the *Name* and *Description* field. 
 . Click *Create*. 
-. For *Token Information*, click *Docker Configuration*. 
-. Enter the following Dockerfile registry authentication code:
+. On the *Token Information* page, click *Docker Configuration* and download or *view its contents* to display the authentication. 
```

Comment:
. Click **Create**. You are taken to the page, *Token Information*. 
. Click **Docker Configuration**. 
. To display the Dockerfile registry authentication code, download the authentication of your new account or click **view its contents**. 
. Enter the Dockerfile registry authentication code. It should look similar to this example:

Reply:
The registry authentication is not a code, so I don't think that part of the change makes sense. The registration is a json object containing a base64 encoded string.

Reply:
Okay, then would the following revision work?:

To display the Dockerfile registry authentication, download the authentication of your new account or click view its contents.
Enter the Dockerfile registry authentication. It should look similar to this example:

So, essentially, simply removing the word `code`